src/components/search/PageSearchModal/index.js

```diff
@@ -84,6 +86,7 @@ module.exports = React.createClass({
       page: page || 1,
       pageSize: this.props.pageSize,
       pageType: this.props.pageType,
+      groupValue: this.props.groupValues
```

Comment:
No. Pages API expects it as `groupValue`, regardless of being an array or string.